### ISSUE TITLE
nh-accuvote: test state ballot conversion changes

### ISSUE THREAD

**Issue by mattroe** [votingworks/vxsuite#2064]
Using the [test ballot fixture](https://github.com/votingworks/vxsuite/blob/main/libs/ballot-interpreter-nh/test/fixtures/amherst-2022-07-12/election.json) as the reference conversion output, we should make the following changes the conversion output for the nh-demo:

1. Remove `WriteIn` as a `centralScanAdjudicationReason` to prevent stopping of write-in ballots on VxCentralScan. (We can alternatively fix VxCentralScan to not show the old write-in adjudication flow if time permits.)
2. Change the ballot style value to something more meaningful than `card-number-3`. This could be as simple as `1` as a hardcoded value for the demonstration. (Future state it should probably be a concatenation of [ElectionID-TownID-PrecinctID].)
3. Populate the full state name for the state value. It currently converts to `NH`. It should be `New Hampshire`.

**Comment by benadida** [comment]
quick question: why do we care about this if the ballots are not printed by us? I know this is low pri, I'm just asking if this should just be closed altogether.

**Comment by mattroe** [comment]
@benadida good point. The only value that shows up in our apps & reports for a non-primary state election is the state name.  Not an issue if it's abbreviated for the demo.
